src/gui/components/tabs/tabs.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { merge } from 'glamor'
+
+import TabsHeader from './tabsHeader'
+import TabsAction from './tabsActions'
+import styles from './tabs.styles'
+
+class Tabs extends Component {
+  state = {
+    selected: this.props.defaultKey,
```

Comment:
I really don't like that pattern.
 - we handle state in graphical components
 - parent give callback, + key to child that call the callback with key as parameter
 - we have a key, this is `title` we don't need to use `index` IMO.

at least, can't we create a curry version of `handleSelectedTab` ?
```es6
handleSelectedTab = (key) => () => {
  //...
}

// ...

titles={titles.map(title => <li {/* ... */} onClick={handleSelectedTab(title)}>{...}</li>)}
```

and to TabsHeader :
```es6
onClick={onSelectTab}
```